frontend/index.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let addClassForm = document.querySelector("#addClass")
+  let addStudentForm = document.querySelector("#addStudent")
+  let listStudentsForm = document.querySelector("#listStudents")
+  let listStudentsInput = document.querySelector("#listStudentsInput") 
+  let classNameInput = document.querySelector("#classNameInput") 
+  let teacherNameInput = document.querySelector("#teacherNameInput") 
+
+  addClassForm.addEventListener("submit", async (e) => {
+    e.preventDefault()
+    let inputValue1 = classNameInput.value
+    let inputValue2 = teacherNameInput.value
+    let url = `http://localhost:3000/class/${inputValue1}/${inputValue2}`
+    axios.post(url).then(res => {
```

Comment:
You can also pass the request like this `axios.post(`http://localhost:3000/class`, {name: inputValue1, teacher: inputValue2})` The object inside the axios request is represented as req.body in your app.js file.